packaging/debian/cvmfs/control

```diff
@@ -1,85 +1,133 @@
-# The file is adjusted in ci/cvmfs/deb.sh
-
 Source: cvmfs
 Section: utils
 Priority: extra
 Maintainer: Jakob Blomer <jblomer@cern.ch>
-Build-Depends: libfuse3-dev (>= 3.3.0), debhelper (>= 9), autotools-dev, cmake, cpio, libcap-dev, libssl-dev, libfuse-dev, pkg-config, libattr1-dev, patch, python3-dev, python3-setuptools, unzip, uuid-dev, valgrind, libz-dev
+Build-Depends:
+ libfuse3-dev (>= 3.3.0),
+ debhelper (>= 9),
+ autotools-dev,
+ cmake,
+ cpio,
+ libcap-dev,
+ libssl-dev,
+ libfuse-dev,
+ libattr1-dev,
+ libz-dev,
+ patch,
+ pkg-config,
+ python3-dev,
+ python3-setuptools,
+ unzip,
+ uuid-dev
 Standards-Version: 3.9.6.1
 Homepage: http://cernvm.cern.ch/portal/filesystem
 
 Package: cvmfs
 Architecture: amd64 armhf arm64
-#Pre-Depends: ${misc:Pre-Depends}   (preparation for multiarch support)
-Depends: cvmfs-config-default | cvmfs-config, gawk, psmisc, lsof, autofs, fuse, curl, attr, libfuse2, zlib1g, gdb, uuid-dev, uuid, adduser, cvmfs-libs (= ${binary:Version}), ${misc:Depends}
-Recommends: cvmfs-fuse3, autofs (>= 5.1.2)
-#Multi-Arch: same   (preparation for multiarch support)
+Depends:
+ cvmfs-config-default | cvmfs-config,
+ cvmfs-fuse3,
+ gawk,
+ psmisc,
+ lsof,
+ attr,
+ zlib1g,
+ gdb,
+ uuid-dev,
+ uuid,
+ adduser,
+ cvmfs-libs (= ${binary:Version}),
+ ${misc:Depends},
+ ${shlibs:Depends}
+Recommends:
+ autofs (>= 5.1.2)
+Suggests:
```

Comment:
I don't think this should be Suggests.  I'm quite sure that libfuse2 should be automatically figured out as a Depends so listing it here won't help.  I'm not sure however if fuse will be automatically figured out though because it is a runtime dependency.  Assuming that's the case, it ought to be listed as a Depends, not Suggests.  Maybe this is an argument for turning the control file into a template where this fuse Depends and the libfuse-dev get removed when not building with fuse2 support.

Reply:
Right, with fuse2 we still need fusermount.